compiler/next/test/resolution/testScopeResolve.cpp

```diff
@@ -456,16 +456,233 @@ static void test8() {
     assert(match.id == outerX->id());
     assert(match.found == InnermostMatch::ONE);
   }
- 
+
   {
     const auto& match = findInnermostDecl(context, innerScope, xStr);
     assert(match.id == innerX->id());
     assert(match.found == InnermostMatch::ONE);
   }
 }
 
+// testing a simple import statement
+static void test9() {
+  printf("test9\n");
+  Context ctx;
+  Context* context = &ctx;
+
+  auto path = UniqueString::build(context, "test9.chpl");
+  std::string contents = "module M {\n"
+                         "  import N;\n"
+                         "  N;\n"
```

Comment:
Either this test should ensure that qualified access is of N.x is possible or a separate test should.  Unless that's not enabled yet?

Reply:
Yeah, the issue is that we don't have general support yet for scope resolving Dot expressions. And the test code is using `findInnermostDecl` which can't handle it either. So I'll make a note for it as future work.

Reply:
(Also, I'm not trying here to cover all cases in these C++ tests - we have the start_test tests for that - I just want enough testing to know that we have the basic functionality).